pipeline.groovy

```diff
@@ -130,6 +130,16 @@ def getSources (name) {
     }
 }
 
+def test_travis_yaml () {
+    def script = 'dub test --compiler=$DC'
+    if (fileExists('.travis.yml')) {
+        def travis_script = sh(script: 'get_travis_test_script', returnStdout: true).trim()
```

Comment:
https://github.com/MartinNowak/dlangci/commit/fe388f742a6fc44c7aa8ec0de76153673f6bd211
The [server](https://github.com/MartinNowak/dlangci/commits/server) is going to be merged when the remaining 2 topics are done.